### ISSUE TITLE
Clones with a hive mind?

### ISSUE THREAD

**Issue by iChun** [iChun/Sync#163]
I know this has been suggested before, but someone has given me a proper kind of light and reason to properly implement this....why not the singularity?

**Comment by NanaLich** [comment]
ummm what?

**Comment by 113820** [comment]
A hive mind is a collective mind that controls multiple bodies or entities, these bodies have no individual identity and have the same thought. iChun probably means that you control multiple shells at the same time.

**Comment by iChun** [comment]
Not really, it wouldn't be very possible with minecraft's current controls,
which is why the issue topic has a question mark in it.

**Comment by 113820** [comment]
What I said was the very definition of a hive mind, but it doesn't fully apply here. What about being able to have them automate certain tasks?

**Comment by iChun** [comment]
Not planned, unfortunately.

**Comment by 113820** [comment]
What kind of implementation can we expect?

**Comment by iChun** [comment]
It's a secret for the time being, too, haha sorry D:

**Comment by Iivaitte** [comment]
You could just make the clones "pets", but you'll need the ability to get them to stop attacking something. Maybe some upgrades to record a set of movements or predetermined actions, perhaps you need a machine to keep them running or they have a battery life of some sort where they have to recharge in an unoccupied shell.

Think of these possibilities.

**Comment by denzuko** [comment]
I can see this being kind of like turtles with swarming and follow the leader or akin to AW2's npcs.. hmm.. entire villages made with clones.

**Comment by Darkmega18** [comment]
Pet/follower AI army of clones with player provided equipment with the hero player syncing to another clone body immediately on death like Talandar/Fenixes hero units in Starcraft 2 Multiplayer commanders? :O
